src/main/java/de/baleipzig/iris/ui/view/base/IBaseSearchNNView.java

```diff
@@ -8,4 +8,5 @@
 public interface IBaseSearchNNView<P extends BaseSearchNNPresenter> extends IBaseView<P> {
     void setSearchResult(List<NeuralNetMetaData> neuralNetMetaDatas);
     void lockSearchResultTable(boolean isLocked);
+    void setBodyContentLayoutMargin(boolean enabled);
```

Comment:
warum als interface-funktion? die verwendest du doch nur unten in der reco-view? brauchst du die im presenter noch mal?